eZ/Publish/Core/Persistence/Legacy/Tests/Content/Location/Gateway/DoctrineDatabaseTest.php

```diff
@@ -258,6 +258,44 @@ public function testMoveHiddenSourceUpdate()
         );
     }
 
+    public function testMoveInvisibleSourceUpdate()
```

Comment:
`eZ/Publish/Core` tests are executed on sqlite only and exist here by mistake. The proper test needs to involve API and should be placed in `\eZ\Publish\API\Repository\Tests\LocationServiceTest`, which enables MySQL and PostgreSQL coverage. Otherwise we might miss something.